fixtures/flight/src/App.js

```diff
@@ -66,6 +81,8 @@ export default async function App({prerender}) {
           </div>
           <Client />
           <Note />
+          <Foo>{dedupedChild}</Foo>
+          <Bar>{Promise.resolve([dedupedChild])}</Bar>
```

Comment:
I had to add this hack because there's an unrelated bug that cause debug info to shift order when the props are delayed.

https://github.com/facebook/react/blob/a4964987dc140526702e996223fe7ee293def8ac/packages/react-client/src/ReactFlightClient.js#L2842